docs/pages/2019_MARVEL_Psik_MaX/notebooks/bandstructure.ipynb

```diff
@@ -138,6 +153,31 @@
     "    **results['seekpath_parameters'].get_dict()))"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "If the pseudopotential family that you downloaded is the same efficiency pseudopotential family, but an older version  (the https://archive.materialscloud.org/file/2018.0001/v1/SSSP_efficiency_pseudos.tar.gz -- note the v1 in the link -- you can use the PwBandStructureWorkChain can be used, by providing a modification of the existing protocol"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# This will take approximately 6 minutes on the tutorial AWS (for Al)\n",
+    "results = launch.run(\n",
+    "    PwBandStructureWorkChain,\n",
+    "    code=code,\n",
+    "    structure=structure_Al,\n",
+    "    protocol=Dict(dict={\n",
+    "        'name':'theos-ht-1.0',\n",
+    "        'pseudo':'SSSP-efficiency-1.0'\n",
```

Comment:
Note that this is not correct, the correct one is above (you have to put `pseudo` inside `modifiers`)